src/Symfony/Component/HttpKernel/HttpKernel.php

```diff
@@ -146,7 +146,7 @@ private function handleRaw(Request $request, int $type = self::MASTER_REQUEST)
         $arguments = $event->getArguments();
 
         // call controller
-        $response = \call_user_func_array($controller, $arguments);
+        $response = \call_user_func_array($controller, $arguments); $line = __LINE__;
```

Comment:
I would put this on the next line using `$line = __LINE__ - 1;` instead. The current code does not comply with our coding standards, and would force to exclude the whole HttpKernel.php file from the CS tooling to avoid having PHP-CS-Fixer breaking it on each run.

Reply:
done